Source/SwiftLintFramework/File+SwiftLint.swift

```diff
@@ -9,9 +9,51 @@
 import SourceKittenFramework
 import SwiftXPC
 
-typealias Line = (index: Int, content: String)
+public typealias Line = (index: Int, content: String)
+
+public typealias Region = (startLine: Int, endLine: Int, disabledRules: [String])
+
+public enum CommandAction: String {
+    case Enable = "enable"
+    case Disable = "disable"
+}
+
+public typealias Command = (CommandAction, String, Int)
 
 extension File {
+    public func regions() -> [Region] {
+        let nsStringContents = contents as NSString
+        let commands = matchPattern("swiftlint:(enable|disable):force_cast")
```

Comment:
the rule capturing was already being done by `NSScanner`, but I had accidentally hardcoded `force_cast` in here as part of my testing.